pode.build.ps1

```diff
@@ -269,7 +269,16 @@ Task Pack -If (Test-PodeBuildIsWindows) Build, {
     if (Test-Path $path) {
         Remove-Item -Path $path -Recurse -Force | Out-Null
     }
-
+    try {
+        # Try to get the Docker version to check if Docker is installed
+        docker --version
+        $global:Docker = $true
+    }
+    catch {
+        # If Docker is not available, exit the task
+        Write-Warning "Docker is not installed or not available in the PATH. Exiting task."
+        $global:Docker = $false
+    }
```

Comment:
It'd be better for this to be within the DockerPack Task, and if if drops into the catch write the warning and `return` - just to keep all Docker related steps together.

something like this, at the top of the DockerPack Task:
```powershell
# check if docker is installed, otherwise skip task
try { docker --version }
catch {
    Write-Warning "Docker is not installed or not available in the PATH. Exiting task."
    return
}
```